fxgl-entity/src/main/kotlin/com/almasb/fxgl/entity/components/TransformComponent.kt

```diff
@@ -41,6 +41,7 @@ class TransformComponent(x: Double, y: Double, angle: Double, scaleX: Double, sc
 
     private val propPositionOriginX = SimpleDoubleProperty(0.0)
     private val propPositionOriginY = SimpleDoubleProperty(0.0)
+	private val propPositionOriginZ = SimpleDoubleProperty(0.0)
 
```

Comment:
Looks good, but please fix spacing. The IDE formatting should be set to *4 spaces, no Tabs*.